### ISSUE TITLE
Add "Begin Backup" or "Begin Restore"

### ISSUE THREAD

**Issue by as-muncher** [redorescue/redorescue#95]
using redo rescue 4.0.0

First off, I want to say that I like that it boots into a live distribution, and that I can turn off wireless internet, because I may not need it, and I don't like extra radiation floating around.

But when I was choosing to do create an image, and I went forward to the next screen, Redo rescue started working already, and I would have liked if the button would have said "Begin backup" or "Begin restore" or whatever the case may be instead of jumping right in unexpectedly.

**Comment by macwinkler1** [comment]
I agree to that what @as-muncher  said... but more in the way that I want to see what is redo rescue really doing!
When the program is on part 2 of 2 ...it looks like doesn't make any progression...says reading Superblock..elapsed 00:00:1
remaining 00:00:00...speed 906.95MB...and this is it....nobody knows when things are done...if they are done...that happens to me with version 3 and 4
